packages/slice-machine/src/modules/loading/types.ts

```diff
@@ -9,4 +9,5 @@ export enum LoadingKeysEnum {
   SAVE_CUSTOM_TYPE = "SAVE_CUSTOM_TYPE",
```

Comment:
Interesting to see some duplicated code between this and `packages/slice-machine/src/modules/modal/types.ts`

Reply:
Do you think this would require a refactor?